CNAM-438: DrugPrescription Event Transformer.

ysebiat opened the pull request.

Anfor commented on src/main/scala/fr/polytechnique/cmap/cnam/etl/transformers/drugprescription/DrugPrescriptionTransformer.scala:
```diff
@@ -0,0 +1,38 @@
+    */
+  def fromDrugs(drugs: List[Event[Drug]]): Event[DrugPrescription] = {
+    val first = drugs.head
+    val a = drugs
```
Why "a" as variable name?

ysebiat committed:
CNAM-438: DrugPrescription Event Transformer.


CNAM-438: Correct tests.


CNAM-438: Rename variable, add prescriptions to FallTransform.
